### DISCUSSION TITLE
Capability to search for a "filename"

### DISCUSSION THREAD

**Discussion by leoak** [arnoudkooi/SN-Utils#143] [Ideas]
Hello,

Unless it is already available in a way I am not aware of, would it make sense to have a feature that will open a targeted filename?

As an admin, often times I find messages in system logs referring to a filename.
For example: I can find this warning:
_com.glide.script.RhinoEcmaError: Cannot convert null to an object.
sys_script_include.52d69ceaeb233100469a20425206feb6.script : Line(65) column(0)_

As an admin, I'd like to quickly open:
__sys_script_include.52d69ceaeb233100469a20425206feb6.script__

Currently, I have to do:
sys_script_include.filter
search for record with the sys_id

Thank you

**Comment by arnoudkooi** [comment]
There is a slashcommand` /<sys_id>` or the alias `/sysid <sys_id>` have you checked those.
The fastest is probably:

- Select the sys_id
- Type in `/\<enter>`
- Wait for new tab to open with the result

Als check these tweets:
https://twitter.com/sn_utils/status/1298999609184776194
https://twitter.com/sn_utils/status/1341332908561362946

**Reply by leoak** [reply]
That's awesome thanks!
I got no response for /sys_id...
But using /<my sys_id> is awesome.

Thank you.

**Comment by arnoudkooi** [comment]
Good, note that the slashcommand is /sysid without the underscore
